includes/legacy/abstract-wc-legacy-order.php

```diff
@@ -454,7 +454,7 @@ public function __get( $key ) {
 			return $this->{"get_{$key}"}();
 		} else {
 			return get_post_meta( $this->get_id(), '_' . $key, true );
-		}
+		} // End if().
```

Comment:
Probaby because of https://github.com/squizlabs/PHP_CodeSniffer/blob/master/CodeSniffer/Standards/Squiz/Sniffs/Commenting/LongConditionClosingCommentSniff.php#L175. I have:
```
boro@bor0:~/dev/WordPress-Coding-Standards$ phpcs -i
The installed coding standards are MySource, PEAR, PHPCS, PSR1, PSR2, Squiz, Zend, WordPress, WordPress-Core, WordPress-Docs, WordPress-Extra and WordPress-VIP
```

Maybe I should disable `Squiz` :)